### PR TITLE
ThorVG: update to v0.11.1

### LINKED ISSUE #82982: SVGs with <mask> in them broke, affecting the display of several editor icons
### Godot version

Godot v4.2.dev6 - Pop!_OS 22.04 LTS - X11 - Vulkan (Forward+) - integrated Intel(R) Graphics (ADL GT2) () - 12th Gen Intel(R) Core(TM) i5-1235U (12 Threads)

### System information

Godot v4.2.dev6 - Pop!_OS 22.04 LTS - X11 - Vulkan (Forward+) - integrated Intel(R) Graphics (ADL GT2) () - 12th Gen Intel(R) Core(TM) i5-1235U (12 Threads)

### Issue description

Our SVG icons that use the `<mask>` element broke.

image

Most likely caused by #82542 but I haven't tested it. Technically, `<mask>` is not required for these SVGs, but it allows for easy clean cutouts which is why I used them here.

### Steps to reproduce

Open Godot and find them

### Minimal reproduction project

N/A

### LINKED ISSUE #83198: SVG icons doesn't show images inside
### Godot version

4.2-dev6

### System information

Godot v4.2.beta1 - Windows 10.0.22621 - Vulkan (Forward+) - integrated AMD Radeon(TM) Graphics (Advanced Micro Devices, Inc.; 31.0.12044.3) - AMD Ryzen 3 5300U with Radeon Graphics (8 Threads) 

### Issue description

Checking changes on the Godot v4.2.dev6, also just downloaded beta1 and the same, I advertised that the icon I used in my addon was not rendered correctly.  The problem was that this SVG contained a raster image inside and some vectors over.  The raster image (body) disappeared from the rendered SVG.

image

The SVG was created with Inkscape, and it was correctly rendered with the last stable release of Godot 4.1.
This is how it is rendered correctly with 4.1.
image

### Steps to reproduce

I attached a minimal project, but, really, you only have to get the SVG from there.
Just put the .svg as the icon of the project, and you will see the bad render even in the project manager window.

### Minimal reproduction project

a minimal project to reproduce

[svgtest.zip](https://github.com/godotengine/godot/files/12884066/svgtest.zip)

### PR DESCRIPTION
**Pull request by capnm**
https://github.com/thorvg/thorvg/releases/tag/v0.11.1

Godot related:
+ [SwEngine] Enhanced the quality of the dash line corners.

image

+ [SVG] Fixed a regression bug related to maskContentUnits userSpaceOnUse/objectBoundingBox.

Fixes #82982  - icons

img02

Fixes #83198  - svg ← png/jpg

### CODE REVIEW

**Comment by fire** [comment]
Waiting for the cicd to build, and a regression test on raster inages like jpg,png in svg

**Comment by capnm** [comment]
Embedded `PNG` image works fine, `JPEG` image has rendering issues:

https://github.com/thorvg/thorvg/issues/1705

**Comment by fire** [comment]
What was reported was that png works, but we are waiting on the jpeg fix on thorvg.

**Comment by fire** [comment]
Upstream made an update, waiting for a release.

**Review by fire — APPROVED** [review]

**Review by akien-mga — APPROVED** [review]

File: thirdparty/thorvg/update-thorvg.sh [hunk 1]
```diff
@@ -54,6 +56,9 @@ rm -rfv ../src/renderer/gl_engine
 # Only svg (+raw) loader is enabled.
```
Comment (akien-mga) [inline thirdparty/thorvg/update-thorvg.sh:56]: That comment is no longer accurate.

Comment (capnm) [reply thirdparty/thorvg/update-thorvg.sh]: corrected

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**